Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -735,7 +850,7 @@ public function testSecondLevelCache(): void
         $this->assertDICDefinitionClass($myEntityRegionDef, '%doctrine.orm.second_level_cache.default_region.class%');
         $this->assertDICDefinitionClass($loggerChainDef, '%doctrine.orm.second_level_cache.logger_chain.class%');
         $this->assertDICDefinitionClass($loggerStatisticsDef, '%doctrine.orm.second_level_cache.logger_statistics.class%');
-        $this->assertDICDefinitionClass($cacheDriverDef, ArrayAdapter::class);
+        $this->assertDICDefinitionClass($cacheDriverDef, CacheProvider::class);
```

Comment:
On second thought, having to change this should probably have alerted us to the  bug. Oh well 🤷‍♂️